lms/templates/sudo/sudo.html

```diff
@@ -1,21 +1,24 @@
 {% extends "main_django.html" %}
-{% load i18n %}
+{% load i18n third_party_auth_links %}
 
 {% block body %}
-<section style="margin: 0 auto; width: 480px; padding: 50px;">
+<section class="sudo" style="margin: 0 auto; width: 480px; padding: 50px;">
     <div class="inner-wrapper">
         <header>
           <h2 style="text-align: center;">{% trans "Confirm Your Password to Access the Instructor Dashboard" %}</h2>
         </header>
         <hr />
-        <div style="margin: 0px auto; width: 218px;">
+        <div style="margin: 0px auto; width: 320px;">
             <form class="sudo-form" method="post">{% csrf_token %}
             {{ form.as_p }}
             <p>
-                <input type="submit" value="Submit" />
+                <input type="submit" style="width: 100%;" value="{% trans 'Submit' %}" >
             </p>
             </form>
         </div>
     </div>
+
+    {% third_party_auth_links %}
+
 </section>
 {% endblock %}
```

Comment:
@dan-f We can only move ```third_party_auth_links.html``` into ```django_sudo_helpers``` because the content of both ```sudo.html```'s is different from each other except ```{% load i18n third_party_auth_links %}``` and ```{% third_party_auth_links %}```.